getting-started-for-developers.adoc

```diff
@@ -88,6 +88,8 @@ You will need:
 * An https://en.wikipedia.org/wiki/Comparison_of_integrated_development_environments#Java[IDE]
 (e.g., https://www.jetbrains.com/idea/download/[IntelliJ IDEA], https://www.eclipse.org/downloads/[Eclipse], etc.)
 
+link:../guides/[*_You can find installation instructions for your operating system here._*]
```

Comment:
For instructions that will walk you through installing Java, Maven, and Git for your operating system and show you how to clone and fork the WildFly Elytron project, take a look at our set up https://wildfly-security.github.io/wildfly-elytron/guides/guides[guides].

Reply:
I think it's fine to have this repeated here (some might like to have everything on one page and might not check out the getting started guide).